src/GPX/Reader.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
 namespace DerickR\GPX;
```

Comment:
What was wrong with that empty line above?

Reply:
Not an issue per se. We've talked about coding standards and among the things vscode picked up, that was one of us. I can roll it back.

Reply:
OK, I think that'd be good. I wonder why VS Code whines about this one! Maybe missing license statement? (Which is really missing, actually).

Reply:
Done.